cdisc_rules_engine/services/define_xml/base_define_xml_reader.py

```diff
@@ -338,7 +346,7 @@ def validate_schema(self) -> bool:
         logger.info(f"Validated Define-XML schema. is_valid={is_valid}")
         return is_valid
 
-    @cache
+    @cached
```

Comment:
Why did you change this caching?

Reply:
Thought that it was cached before somewhere and forgot to look for the cached name. Changed it back.